src/decima/cli/vep.py

```diff
@@ -121,3 +122,27 @@ def cli_predict_variant_effect(
         gene_col=gene_col,
         genome=genome,
     )
+
+
+@click.command()
```

Comment:
Instead of making a separate CLI command to ensemble vep predictions, I would suggest making this a part of the 'vep' command, to run only if the user supplies model = 'ensemble'.